routes.test.js

```diff
@@ -0,0 +1,89 @@
+import { Cookie } from 'tough-cookie';
+import { Observable } from 'rxjs';
+import Hapi from 'hapi';
+import getRoutes from './routes';
+import getConfig from './util/config';
+
+const server = new Hapi.Server();
+
+server.connection();
+
+// mock the anonAuthPlugin
+server.decorate(
+	'request',
+	'authorize',
+	request => () => Observable.of(request),
+	{ apply: true }
+);
+
+const MOCK_RESULT = 'asdf';
+const MOCK_RENDER_REQUEST = () => Observable.of({ result: MOCK_RESULT, statusCode: 200 });
+
+const MOCK_renderRequestMap = {
+	'en-US': MOCK_RENDER_REQUEST,
+};
+
+const MOCK_OAUTH_COOKIES = {
+	oauth_token: '1234',
+	refresh_token: 'asdf',
+	anonymous: true
+};
+
+const MOCK_API_RESULT = {
+	queries: [],
+	responses: [],
+};
+
+const serverReady = getConfig()
```

Comment:
Unrelated to this PR, but this is the first time i'm noticing that `config.js` is a Promise based interface. Any plans on using observables here instead?